README.md

```diff
@@ -117,18 +117,18 @@ Take a look at [doc.bunq.com](https://doc.bunq.com) for the full documentation.
 BunqContext::loadApiContext($apiContext); // if it has not been loaded yet. 
 
 Payment::create(
-            new Amount($amount, self::CURRENCY_TYPE_EUR),
-            new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
-            $description,
-            $monetaryAccount->getId()
-        )
+    new Amount($amount, self::CURRENCY_TYPE_EUR),
+    new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
+    $description,
+    $monetaryAccount->getId()
+)
 ```
 
 ##### Example
 See [`tinker/BunqLib`](https://github.com/bunq/tinker_php/blob/05a38a2660e6f6db1f7efc9b915f0131c172c230/src/BunqLib.php#L240-L245)
 
 #### Reading objects
-To use the read method you must pass the identifier of the object to read (ID or UUID) expect for the endpoints User, UserPerson, UserCompany and monetary account. As the SDK will use default ids if not passed. For all other endpoints you must pass the identifier. 
+To use the read method you must pass the identifier of the object to read (ID or UUID) except for the endpoints `User`, `UserPerson`, `UserCompany` and `Monetary Account`. The SDK will use the default IDs when none are passed. For all other endpoints you must pass the identifier.
```

Comment:
We refer to `UserPerson` and `UserCompany` then I would also refer to `MonetaryAccount`